Valigator.AspNetCore/IDescriptor.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Valigator.Core;
+
+namespace Valigator
+{
+	public interface IDescriptor
```

Comment:
IDescriptorProvider? Or something like that. The current name makes it sound like it is a descriptor.